pep-9999.rst

```diff
@@ -0,0 +1,259 @@
+PyPA.
+
+Processes
+=========
+
+The processes for the PyPAs activities are outlined below:
+
+Specifications
+--------------
+
+The PyPA will use PEPs for defining, and making changes to, the
+interoperability specifications maintained by the PyPA. Thus, the
+Python Steering Council has the final say in the acceptance of these
+interoperability specifications.
+
+It is expected (but not required) that the Python Steering Council
+delegates the authority to approve or reject PEPs to individuals
+within the PyPA community. At the time of writing (November 2019), the
+Python Steering Council has `standing delegations`_ for currently
+active packaging interoperability specifications.
+
+The details of the process of proposing and updating the
+interoperability specifications are described in the `PyPA
+Specifications`_ document.
+
+Governance
+----------
+
+PyPA Committer Votes
+''''''''''''''''''''
+
+A PyPA member can put forward a proposal and call for a vote on a
+public PyPA communication channel. A PyPA committer vote is triggered
+when a PyPA committer (not the proposer) seconds the proposal.
+
+The proposal will be put to a vote on the `PyPA-Committers`_ mailing
+list, over a 7 day period. Each PyPA committer can vote once, and can
+choose one of ``+1`` and ``-1``. If at least two thirds of voters vote
+``+1``, then the vote succeeds.
+
+PyPA committer votes are required for, and limited to, the following
+kinds of proposals:
+
+Addition of a project to the PyPA
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Proposing the acceptance of a project into the PyPA organization. This
+proposal must not be opposed by the existing maintainers of the
+project.
+
+Creation of a new project in the PyPA
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Proposing the creation of a new tools / project in the PyPA
+organization.
+
+Removal of a project from PyPA
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Proposing the removal of a project in the PyPA organization.
+
+Updates to the Governance/Specification Processes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Proposing changes to how the PyPA operates, including but not limited
+to changes to its specification and governance processes, and this
+PEP.
+
+Leaving PyPA
+''''''''''''
+
+A project that is a part of the PyPA organization, can request to
+leave PyPA.
+
+Such requests can made by a committer of the project, on the
+`PyPA-Committers`_ mailing list and must clearly state the GitHub
+user/organization to transfer the repository to.
+
+If the request is not opposed by another committer of the same project
+over a 7 day period, the project would leave the PyPA and be
+transferred out of the PyPA organization as per the request.
+
+References
+==========
+
+.. _PyPI: https://pypi.org
+.. _PyPA Organization: https://github.com/pypa
+.. _PyPA Specifications: https://www.pypa.io/en/latest/specifications
+.. _PyPA-Committers: https://mail.python.org/mm3/mailman3/lists/pypa-Committers.python.org/
```

Comment:
Capital "C" breaks the link: https://mail.python.org/mailman3/lists/pypa-committers.python.org/